pkg/report/linux.go

```diff
@@ -974,7 +974,7 @@ var linuxOopses = append([]*oops{
 						parseStackTrace,
 					},
 					// These frames are present in KASAN_HW_TAGS reports.
-					skip: []string{"kernel_fault", "tag_check", "mem_abort", "el1_abort", "el1_sync"},
+					skip: []string{"kernel_fault", "tag_check", "mem_abort", "el1_", "el1h_"},
```

Comment:
Wouldn't it be more generic than we want?

There are methods named `ip6_mc_del1_src`, `hpsa_free_ioaccel1_cmd_and_bft`, `channel1_intr_enable`, etc. which all contain `el1_` as a substring.

Maybe it makes more sense to require that the token either starts with `el1_` or that it's actually the `_el1_` substring.

Reply:
You are totally right.
But I am not sure how to spot fix this. These are matched as substring, so it's not possible to express "^el1_" at the moment.
There also may be more of similar cases because I used to be a bit sloppy with these strings.
If you don't see an easy way to fix this, what do you think if we merge this and then I will ask you to change matching to use regexps and then change "el1_" and maybe some other prefixes? :)

Reply:
Turns out regexps just work. Fixed. I've also added ip6_mc_del1_src frame to the report for testing purposes.
PTAL.